assets/css/general.css

```diff
@@ -0,0 +1,4 @@
+body {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
```

Comment:
Fix this as well. Check this comment https://github.com/hedita/Acme-web-Solutions/pull/1/files#r785486933

Reply:
I cannot see any commit regarding to fixing this. Just to clarify this, you need to have the related font family only here. You are not using `Roboto`, etc. I added all `Open Sans` here but look at the `font-family`.
![image](https://user-images.githubusercontent.com/1394580/149747875-6d60b6f0-0ad0-4ebf-9d6f-773b28244324.png)

Reply:
I cannot see any commit regarding to fixing this. Just to clarify this, you need to have the related font family only here. You are not using `Roboto`, etc. I added all `Open Sans` here but look at the `font-family`.

![image](https://user-images.githubusercontent.com/1394580/149748080-3ad6d74d-5321-495c-8c80-749903527418.png)